### PR TITLE
docs: updated all P components doc

### PR DESCRIPTION
**Pull request by matthprost**
## Type

- Documentation

#### What is expected?

Refacto documentation of all P components:

- PasswordCheck
- PasswordStrengthMeter
- Pentagon
- PhoneInput
- Placeholder
- Popper
- Progress
- ProgressBar
- ProgressCircle
- ProgressionButton

### CODE REVIEW

File: src/components/Progress/index.js [hunk 1]
```diff
@@ -90,7 +90,7 @@ const StyledStep = styled(Box, {
   ${({ clickable }) => clickable && clickableStyles}
 `
 
-const Step = () => null
+export const Step = () => null
```
Comment (chambo-e) [inline src/components/Progress/index.js:93]: If there is no props there is no value to export it no ?

**Comment by DorianMaliszewski** [comment]
No API on Pentagon :'(

File: src/components/Progress/__stories__/index.stories.mdx [hunk 2]
```diff
@@ -51,5 +61,8 @@ import Button from '../../Button'
   </Story>
 </Canvas>
 
+## API
+
+### `Progress`
+
 <ArgsTable of={Progress} />
```
Comment (DorianMaliszewski) [inline src/components/Progress/__stories__/index.stories.mdx:68]: Missing Progress.Step API ?

File: src/components/ProgressBar/__stories__/index.stories.mdx [hunk 3]
```diff
@@ -1,7 +1,12 @@
-import { Canvas, Description, Meta, Story } from '@storybook/addon-docs/blocks'
-import { useEffect, useState } from 'react'
+import {
```
Comment (DorianMaliszewski) [inline src/components/ProgressBar/__stories__/index.stories.mdx:1]: Sidebar order !== Story order and Titles are differents

**Review by DorianMaliszewski — CHANGES_REQUESTED** [review]
Lot of component have differences between story order/name and sidebar

File: src/components/Placeholder/__stories__/index.stories.mdx [hunk 4]
```diff
@@ -67,6 +84,6 @@ import Placeholder from '..'
   </Story>
 </Canvas>
 
-## Props
+## API
```
[the remark this answers is not in the document]
Comment (matthprost) [reply src/components/Placeholder/__stories__/index.stories.mdx:87]: I think so but it implies a migration

[the remark this answers is not in the document]
Comment (DorianMaliszewski) [reply src/components/Placeholder/__stories__/index.stories.mdx:87]: Yeah

File: src/components/Placeholder/__stories__/index.stories.mdx [hunk 5]
```diff
@@ -1,10 +1,27 @@
-import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs/blocks'
+import {
```
[the remark this answers is not in the document]
Comment (matthprost) [reply src/components/Placeholder/__stories__/index.stories.mdx:1]: BoxWithIcon = box

File: src/components/Progress/index.js (hunk 1 above)
Comment (matthprost) [reply src/components/Progress/index.js:93]: I forgot to add proptypes but there is a one prop

File: src/components/Placeholder/__stories__/index.stories.mdx (hunk 4 above)
[the remark this answers is not in the document]
Comment (matthprost) [reply src/components/Placeholder/__stories__/index.stories.mdx:87]: @DorianMaliszewski fixed

**Review by DorianMaliszewski — CHANGES_REQUESTED** [review]
Still no API in Pentagon : 
Capture d’écran 2021-06-01 à 16 00 40

Still bad order in PhoneInput : 
Capture d’écran 2021-06-01 à 16 01 20

Same in Popper : 
Capture d’écran 2021-06-01 à 16 02 02

Same In Progress : 
Capture d’écran 2021-06-01 à 16 02 26
Capture d’écran 2021-06-01 à 16 02 48

Stil in ProgressBar : 
Capture d’écran 2021-06-01 à 16 03 14


ProgressCircle : 
Capture d’écran 2021-06-01 à 16 03 40

**Review by chambo-e — APPROVED** [review]

**Review by DorianMaliszewski — APPROVED** [review]

**Pull request merged**